lib/[celeb]/index.tsx

```diff
@@ -7,23 +8,37 @@ import { Md } from '~/lib/[celeb]/Md/Md';
 import { PageHead } from '~/lib/[celeb]/PageHead/PageHead';
 import { TagCollection } from '~/lib/[celeb]/TagCollection';
 import { TopSection } from '~/lib/[celeb]/TopSection/TopSection';
+import { useCeleb } from '~/lib/[celeb]/useCeleb';
+import { AppBar } from '~/lib/_app/AppBar/AppBar';
 
 export const Celeb = (p: CelebPageProps) => {
+  useCeleb(p.celeb.name, p.celeb.facts);
+
   return (
-    <main>
-      <PageHead />
+    <>
+      <AppBar />
 
-      <TopSection />
+      <main className="flex flex-col bg-gray-100">
+        <div className="mx-auto max-w-4xl">
```

Comment:
I think the app looks better at 4xl max than 5xl max. Let's discuss it if you disagree.